manifests/install.pp

```diff
@@ -26,9 +26,8 @@
     ensure => directory,
   }
 
-  file { $nexus::data_path:
+  file { $nexus::install_path:
```

Comment:
This directory will be created on line 16, because the `managehome => true`, so this `file` resource looks unnecessary. If the `$nexus::data_path` will not be managed (here?), the resource can be removed.